src/main/java/dataMgmt/SessionStorage.java

```diff
@@ -40,6 +68,57 @@ public void setAuthenticationToken(String authenticationToken) {
     }
 
     /**
+     * Get the current user's loaded projects.
+     * @return projects list
+     */
+    public List<Project> getProjects() {
+        List<Project> projects = new ArrayList<>(this.projects.values());
+        Collections.sort(projects, (o1, o2) -> {
```

Comment:
Since the implementation of the storage of the projects is now a hashmap, we wanted the getProjects() function to be consistent in the order in which it returns the projects, since it is still only exposing the projects as a list.